io/net/ethtool_test.py

```diff
@@ -70,6 +71,16 @@ def interface_state_change(self, interface, state, status):
         cmd = "ip link set dev %s %s" % (interface, state)
         if process.system(cmd, shell=True, ignore_status=True) != 0:
             return False
+        if state == "up":
+            for _ in range(0, 120, 10):
```

Comment:
Can we use wait module in avocado utils to wait for function to return true 
wait.wait_for(check_state(interface), 30)

as per discussion with murilo a 30 sec wait is sufficient for any adapter link to come up

Reply:
@abdhaleegit this has been tested...  30 sec is an random assumtions.  tg3 driver takes from 20 sec to 2 min to complete based on what ethtool parameter is changed.

Reply:
btw wait module is not fully functional!!. so cant use it.

Reply:
the logic checks the status of interface from 10 sec to 2 min. so there is no full wait of the system indefintly or some random 30 sec waiting .

Reply:
Abdul,  Let me know if you ok with the justification

Reply:
@manvanthar Also, if this is a generic solution, can this be pushed to avocado utils ?

@abdhaleegit your comments ?

Reply:
the wait.wait_for is used to wait for command to complete .. but the logic here is really waiting till the network link is up.. so we can go ahead with this logic.. in future a avocado util to check link up will be better.

Reply:
we can push this till than